### ISSUE TITLE
Seniorság, vezető seniorság

### ISSUE THREAD

**Issue by vbalazs** [kir-dev/simonyi-cimtar#34]
Kellenek ilyen globális posztok.
Ki kell találni hogyan tároljuk, mit tudjanak csinálni, ki tudja osztani ezt a tisztet.

**Comment by vbalazs** [comment]
akkor ez legyen egy jogosultságok nélküli (üres) role?

**Comment by tmichel** [comment]
egyrészt lehet az is. és akkor megadjuk a lehetőséget arra hogy a jövőben nem legyen para kibővíteni, ha mégis akarnak nekik valami rendszer szintű jogosultságot.

a másik megoldás az lehet, hogy lesz egy gomb, hogy az adott tagot vezető seniornak nevezzük ki. és akkor semmi role nincs, csak létrehozunk egy rekordot a posts táblában egy előre beállított névvel.

**Comment by vbalazs** [comment]
ha role akkor global role kellene, de ahogy látom azok nincsenek naplózva a történetben, mert nincs membership. ez szandekos?

**Comment by tmichel** [comment]
igen. logikus, hogy az egy körön belüli posztokat kell csak felvenni a history-ba. nincs értelme az egységhez (körhöz) nem tartozó role-król is feljegyezni, hogy ki mikor birtokolta.

ebben az esetben viszont a globális role teória ugrik. viszont akkor beleütközünk abba a problémába, hogy egy post egy `Membership`-hez van kötve. tehát ha globális postokat szeretnénk akkor vissza a startvonalhoz és újra kell gondolni.

**Comment by vbalazs** [comment]
ne írj ilyeneket :D 
nem biztos egyébként, hogy ezt a dolgot ebbe kellene szuszakolni. oké, hogy vannak már viszonylag külön élő szerepek és posztok, de én simán elképzelhetőnek tartok egy ezektől külön élő globális, profilhoz kötött posztokat is. ilyen pl. a nevtanár (ez most jelenleg a vezetoseg egysegben egy egyseg szintu posztkent van nyilvantartva), senior, nev. senior

**Comment by tmichel** [comment]
hm. ez nem olyan rossz gondolat. azt is lehet akkor, hogy a `Post` valójában tartozhat egy memberhez is a membership mellett. van erre [polymorphic association](http://guides.rubyonrails.org/association_basics.html#polymorphic-associations)-nek nevezett okosság

**Comment by vbalazs** [comment]
> hm. ez nem olyan rossz gondolat. azt is lehet akkor, hogy a Post valójában tartozhat egy memberhez is a 
> membership mellett. van erre polymorphic association-nek nevezett okosság

biztos, hogy jó ez nekünk? egy globalPost szerintem karbantarthatóbb és könnyebben átláthatóbb lenne.
